Plots/Overlays/NCL_overlay_6.py

```diff
@@ -209,13 +210,15 @@
                   transform=ccrs.PlateCarree(),
                   headwidth=5,
                   cmap=wind_cmap,
-                  norm=norm,
+                  # norm=norm,
```

Comment:
Are you planning on leaving this line in as a comment, or do you plan to remove it all together? It's a bit ambiguous.